### PR TITLE
feat: create newsletter page

### PR DESCRIPTION
**Pull request by Bmullerc**
## Why

Kassandra needs a newsletter page.

## Changes

- [x] New feature (non-breaking change which adds functionality)
  1. This PR adds a newsletter page with placeholder content.

### CODE REVIEW

File: src/pages/newsletter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Newsletter } from '@/templates/Newsletter'
+import Head from 'next/head'
+
+export default function Index() {
+  return (
+    <>
+      <Head>
+        {/* Primary Meta Tags */}
+        <title>Kassandra Newsletter</title>
+        <meta name="description" content="" />
```
Comment (nicholascostadev) [inline src/pages/newsletter/index.tsx:10]: Ask someone for a description

File: src/pages/newsletter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+        {/* Open Graph / Facebook */}
+        <meta property="og:url" content="https://kassandra.finance/" />
+        <meta property="og:description" content="" />
```
Comment (nicholascostadev) [inline src/pages/newsletter/index.tsx:14]: Here too

File: src/pages/newsletter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+        {/* Twitter */}
+        <meta property="twitter:url" content="https://kassandra.finance/" />
+        <meta property="twitter:description" content="" />
```
Comment (nicholascostadev) [inline src/pages/newsletter/index.tsx:18]: Here too

**Review by nicholascostadev — APPROVED** [review]

File: src/pages/newsletter/index.tsx (hunk 3 above)
Comment (Bmullerc) [reply src/pages/newsletter/index.tsx:18]: I was instructed to leave it blank

File: src/pages/newsletter/index.tsx (hunk 2 above)
Comment (Bmullerc) [reply src/pages/newsletter/index.tsx:14]: I was instructed to leave it blank

File: src/pages/newsletter/index.tsx (hunk 1 above)
Comment (Bmullerc) [reply src/pages/newsletter/index.tsx:10]: I was instructed to leave it blank

**Review by ChristanDaniel — APPROVED** [review]

**Pull request merged**